src/context_store/storage/sqlite.py

```
@@ -469,13 +490,41 @@ async def delete_memory(self, memory_id: str) -> bool:
+                    meta_row = None
                 async with conn.execute(
                     "SELECT id FROM memories WHERE id = ?", (memory_id,)
                 ) as cursor:
                     exists = await cursor.fetchone()
                 if exists is None:
                     return False
                 await conn.execute("DELETE FROM memories WHERE id = ?", (memory_id,))
+                if self._outbox_writer is not None:
+                    try:
+                        tags = json.loads(meta_row[1]) if meta_row and meta_row[1] else []
+                    except Exception:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)